sormas-api/src/main/java/de/symeda/sormas/api/contact/ContactIndexDto.java

```diff
@@ -40,6 +40,7 @@
 	public static final String DISEASE = "disease";
 	public static final String REGION_UUID = "regionUuid";
 	public static final String DISTRICT_UUID = "districtUuid";
+	public static final String COMMUNITY_UUID = "communityUuid";
```

Comment:
Please move these constants to the ContactJurisdictionDto. It's also used in getPrefixCaption in some places. There I'd suggest to do e.g. `I18nProperties.getPrefixCaption(ContactJurisdictionDto.I18N_PREFIX, ContactJurisdictionDto.REGION_UUID)`. ContactJurisdictionDto can use the same I18n_PREFIX as ContactIndexDto.